utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -841,36 +855,21 @@ int main(int argc, char **argv)
                     else
                         missingFuncFlag = 1;
 
-                    break;
-                }
-                case 45:
```

Comment:
Why have we touched all this? Pls revert. We just need to add case 5 for pixelate

Reply:
reverted back